.github/workflows/scorecard.yaml

```diff
@@ -0,0 +1,54 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '37 10 * * 0'
```

Comment:
Can we change the schedule time to `00 01 * * 0` every Sunday at 01 AM ?

Reply:
done